rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -231,12 +235,18 @@ def get_image_id(self, array):
     def get_bounding_box_count(self):
         return b.getBoundingBoxCount(self._handle)
 
-    def get_bounding_box_abels(self):
+    def get_bounding_box_labels(self):
         return b.getBoundingBoxLabels(self._handle)
 
     def get_bounding_box_cords(self):
         return b.getBoundingBoxCords(self._handle)
 
+    def get_mask_count(self, array):
```

Comment:
Add all the new functions at the end at a single place

Reply:
I've added the metadata functions together and the info functions (img_sizes()) together to make it easier to group functions